### PR TITLE
Support for LDAP dynamic DB

### PR DESCRIPTION
**Pull request by anoother**
Conflicts:
    data/osfamily/Debian.yaml
    manifests/params.pp

### CODE REVIEW

File: data/osfamily/Debian.yaml [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
-bind::defaults::confdir: '/etc/bind'
-bind::defaults::namedconf: '/etc/bind/named.conf'
-bind::defaults::cachedir: '/var/cache/bind'
+bind::params::supported: true
+bind::params::bind_user: 'bind'
+bind::params::bind_group: 'bind'
+bind::params::bind_package: 'bind9'
+bind::params::bind_service: 'bind9'
+bind::params::nsupdate_package: 'dnsutils'
+bind::params::dyndb_ldap_package: 'bind9-dyndb-ldap'
```
Comment (inkblot) [inline data/osfamily/Debian.yaml]: `bind::params` does not exist. This looks like a bad merge resolution.

File: templates/dynamicdb/ldap.conf.erb [hunk 2]
```diff
@@ -0,0 +1,52 @@
+
+# This file managed by puppet - changes will be lost
+dynamic-db "<%= @dbname %>" {
+	library "ldap.so";
```
Comment (inkblot) [inline templates/dynamicdb/ldap.conf.erb]: `ldap.so` seems like the kind of thing that will vary by platform. This is fine for now, though.

File: manifests/params.pp [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# ex: syntax=puppet si ts=4 sw=4 et
+
+class bind::params (
```
Comment (inkblot) [inline manifests/params.pp]: NO.

Comment (anoother) [inline manifests/params.pp]: Yes, I see why this is an issue. Not sure why we've done this, I will revise this request.

**Comment by inkblot** [comment]
Could you please do the revision based on version 5.2.0 which I have just released to the Puppet Forge?

**Comment by inkblot** [comment]
There was a bug in 5.2.0, and 5.2.1 is released now.

**Pull request closed without merging**